lib/pull_request_description_service.rb

```diff
@@ -34,7 +34,7 @@ def run
     # @return [String] the generated PR description
     def generate_prompt
       # If the pull_request_template.md file is set, read its contents
-      template_markdown = if @pull_request_template != ""
+      template_markdown = if @pull_request_template && !@pull_request_template.empty?
```

Comment:
This can simply be `@pull_request_template&.present?`

Reply:
I wanted to but  `.present?` is only a rails method.